r2/r2/models/subreddit.py

```diff
@@ -553,6 +553,20 @@ def remove_moderator(self, user, **kwargs):
     def moderators(self):
         return self.moderator_ids()
 
+    @property
+    def top_moderator_account(self):
+        if len(self.moderators) >= 1:
+            return Account._byID(self.moderators[0])
+        else:
+            return FakeAccount()
```

Comment:
Just realized you guts may want this line to actually return `None` instead of a `FakeAccount()`